great_expectations/expectations/expectation_configuration.py

```diff
@@ -152,6 +152,7 @@ def __init__(  # noqa: PLR0913
         expectation_type: str,
         kwargs: dict,
         meta: Optional[dict] = None,
+        notes: str | list[str] | None = None,
```

Comment:
Don't love updating this object but we do some translation between expectation and config so we need parity